tests/cors.rs

```diff
@@ -92,8 +118,10 @@ fn success() {
         "https://hyper.rs"
     );
     assert_eq!(res.headers()["access-control-allow-credentials"], "true");
-    let headers = &res.headers()["access-control-allow-headers"];
-    assert!(headers == "x-bar, x-foo" || headers == "x-foo, x-bar");
+    let allowed_headers = &res.headers()["access-control-allow-headers"];
+    assert!(allowed_headers == "x-bar, x-foo" || allowed_headers == "x-foo, x-bar");
+    let exposed_headers = &res.headers()["access-control-expose-headers"];
+    assert!(exposed_headers == "x-header1, x-header2" || allowed_headers == "x-header2, x-header1");
```

Comment:
CI noticed the tests fail, likely because this is checking `allowed_headers` instead of `exposed_headers`.